### PR TITLE
Add 'validate' method

### PR DESCRIPTION
**Pull request by talyguryn**
Prevent from saving with empty fields

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@editorjs/attaches",
-  "version": "1.2.0",
+  "version": "1.3.0",
```
Comment (neSpecc) [inline package.json]: i think, it should be a patch, because this PR actually is a regular bug fix

**Comment by talyguryn** [comment]
resolves #46 

image

File: package.json [hunk 2]
```diff
@@ -24,19 +24,19 @@
...
     "@codexteam/ajax": "^4.0.1",
     "babel-loader": "^8.2.2",
-    "css-loader": "^5.2.4",
-    "eslint": "^6.1.0",
-    "eslint-config-codex": "github:codex-team/eslint-config",
-    "eslint-loader": "^2.2.1",
+    "css-loader": "^1.0.0",
+    "eslint": "^7.32.0",
+    "eslint-config-codex": "^1.6.4",
+    "eslint-webpack-plugin": "^3.1.1",
```
Comment (neSpecc) [inline package.json:30]: Does it work?

Looking on the [official doc](https://webpack.js.org/plugins/eslint-webpack-plugin/):

> This is eslint-webpack-plugin 3.0 which works only with webpack 5. For the webpack 4, see the 2.x branch.

Comment (talyguryn) [reply package.json:30]: yep. works good

need to get the latest version cause

image

**Comment by neSpecc** [comment]
Pls link resolved issues

**Comment by talyguryn** [comment]
resolves #46
resolves #48

**Review by neSpecc — APPROVED** [review]

**Review by ilyamore88 — APPROVED** [review]

**Pull request merged**